models/marts/core/fct_ga4__pages.sql

```diff
@@ -9,7 +9,6 @@ with page_view as (
         count(distinct user_key ) as users,
         sum( if(ga_session_number = 1,1,0)) as new_users,
         sum(entrances) as entrances,
-        sum(exits) as exits,
```

Comment:
Why drop this? I don't personally love it as a metric, but people tend to ask for whatever they see is missing from other reports and I think it's easier to have the metric than to explain why there are better things to look at.